mount/src/assets/user.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import SortIcon from '@mui/icons-material/Sort';
+
+
+
+export default class Users extends React.Component {
+
+  render() {
+    const users = this.props.users;
+    return (
+      <div>
```

Comment:
wrap with <Fragment> if no css added in html tag , as <div> create a extra node in DOM  and fragment doesn't